CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/BlobRequestHandler.java

```diff
@@ -37,6 +44,39 @@ public InputStream createImageContent(Args args) throws IOException {
         return RequestHandlerDispatcher.context.getAsset(imgFilePath[imgFilePath.length - 1]);
     }
 
+    public byte[] createImageContent(Args args) throws IOException {
+        String filePath = args.get("image");
+        if (filePath == null || filePath.isEmpty()) {
+            throw new IOException("Image content file path cannot be null");
+        }
+
+        String[] imgFilePath = filePath.split("/");
```

Comment:
Instead of parsing the string, you should probably use File.getName()

... also... I'm curious as to why this is necessary.  Why are you throwing out most of the path?